src/main/webapp/app/modules/administration/user-management/user-management-detail.tsx

```diff
@@ -24,82 +24,80 @@ export class UserManagementDetail extends React.Component<IUserManagementDetailP
         <h2>
           <Translate contentKey="userManagement.detail.title">User</Translate> [<b>{user.login}</b>]
         </h2>
-        <Row size="md">
+        <dl className="jh-entity-details">
+          <dt>
+            <Translate contentKey="userManagement.login">Login</Translate>
+          </dt>
+          <dd>
+            <span>{user.login}</span>
```

Comment:
I wonder if user is null on first render. Consider adding some validation above (user && ...).

Reply:
This part is generated from jhipster and there was never a problem with this one